### PR TITLE
Feature/jls 6158463071/new dta content

### PR DESCRIPTION
**Pull request by ecstech-admin**
## What type of PR is this? (check all applicable)

- [x] Feature
- [ ] Bug Fix
- [ ] Hot Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Substantial content updates per Word Doc provided by Alec in Monday Task #6158463071

## Related Tickets & Documents
- Monday Task #6158463071

## QA Instructions, Screenshots, Recordings
PR for review of Diffusion Talent Accelerator page

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_
- [x] Semantic HTML implemented?
- [ ] Keyboard operability supported?
- [x] Checked with [Google Lighthouse](https://developer.chrome.com/docs/lighthouse/overview) and addressed `Critical` and `Serious` issues?
- [x] Color contrast tested?

## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized

### CODE REVIEW

File: _site/diffusion-talent-accelerator/index.html
Comment (ronald-bbg) [inline _site/diffusion-talent-accelerator/index.html:1]: For alt text best practices, it is recommended that we don't need to mention "Photo of..." for photos. We can start for e.g. "A woman speaking..." as it satisfies descriptive text.

File: diffusion-talent-accelerator/index.html [hunk 1]
```diff
@@ -19,66 +19,19 @@
...
-    </section>
-
-    <section id="secondary-banner" class="secondary-banner">
-        <div class="secondary-banner__content-wrapper">
-            <div class="grid-container">
-                <div class="grid-row grid-gap">
-                    <div class="grid-col-12">
-                        <div class="secondary-banner__wordmark text-center">
-                            <h2>Diffusion Talent Accelerator Cohort 1 is underway.</h2>
-                            <div class="secondary-banner__tagline">Diffusion of Excellence will be recruiting Cohort 2 in January of 2024. <br />For more information, please email <a href="mailto:VACODiffusionAccelerator@va.gov">VACODiffusionAccelerator@va.gov</a>.</div>
```
Comment (ronald-bbg) [inline diffusion-talent-accelerator/index.html:71]: The copy should be updated to : Recruitment is currently closed. Applications for the next cohort will open
in January 2025. For more information, please email
VHADiffusionTalentAccelerator@VA.gov. (Alec's doc for reference)

**Review by ronald-bbg — CHANGES_REQUESTED** [review]
Screenshot 2024-03-15 at 11 21 06 AM
Not sure if intentional but see a faint gray border where the arrow images are placed.


Screenshot 2024-03-15 at 11 22 22 AM

Should we also have even margin or padding above and below the "Empower your impact section"?

**Comment by ecstech-admin** [comment]
Updates made to PR:

- Timeline: new 508 contrast colors for backgrounds
- Timeline: add margin bottom
- Timeline: rm right-left margins
- Adjust alt tags per Ronalds' comment

**Review by RyanCrux — CHANGES_REQUESTED** [review]
I went though the Monday ticket Word doc line by line. 2 fixes needed. Everything else looks great Jim. 

 1) It looks like the text here is white:
Screenshot 2024-03-18 at 2 33 49 PM


2) Something is missing/ design does not match on the DTA timeline:
Screenshot 2024-03-18 at 2 34 00 PM

**Review by RyanCrux — APPROVED** [review]

**Pull request merged**